apps/everest/src/hooks/api/backup-storages/useBackupStorages.ts

```diff
@@ -37,6 +37,14 @@ export const useBackupStorages = () =>
     queryFn: getBackupStoragesFn,
   });
 
+export const useBackupStoragesByNamespace = (namespace: string) =>
+  useQuery<GetBackupStoragesPayload, unknown, BackupStorage[]>({
+    queryKey: [BACKUP_STORAGES_QUERY_KEY],
+    queryFn: getBackupStoragesFn,
+    select: (data) =>
+      data.filter((item) => item.allowedNamespaces.includes(namespace)),
```

Comment:
Maybe we can add this check here in case NoAllowedNamespaces bug appears again so it doesn't break whole app
```suggestion 
      data.filter((item) => item.allowedNamespaces?.includes(namespace)),
```